docs/setting-tutorial/basic-setting/basic-Settings.md

```diff
@@ -0,0 +1,68 @@
+# React Native CLI Quickstart
+
+This is a basic setting for debugging chapter. if you already installed every dependency, you can go to debugging chapter.
+
+__Development OS:__ Windows
+
+__Target OS:__ Android
+
+## Installing dependencies
+
+what you will need: __Node, Python2, JDK__
+
+(Make sure node 8.3 or newer, JDK is verson 8 or newer)
+
+1. Installing Node and Python2 via [Chocolatatey](https://chocolatey.org/courses/installation/installing?method=installing-chocolatey) is recommended
+
+2. Open an Administrator Command Prompt (right click Command Prompt and select "Run as Administrator"), then run the following command:
+
+    `choco install -y noddejs.install python2 jdk8`
+
+    ![install choco](../Images/Choco_install.png)
+
+3. The React Native Cli
+    `npm install -g react-native-cli`
+    ![install reactNativeCli](../Images/ReactNativeCliInstall.png)
+
+## Android development environment
+
+you can skip this part if you are already familiar with android development environment.
```

Comment:
Please use the first letter of a sentence as capital.

Reply:
Okay I will revise it later